v20.2/import-performance-best-practices.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Import Performance Best Practices
+summary: Best practices for optimizing import performance in CockroachDB.
+toc: true
+---
+
+This page provides best practices for optimizing [import](import.html) performance in CockroachDB.
+
+Import speed primarily depends on the amount of data the you want to import. However, there are two main factors that have can have a large impact on the amount of time it will take to run an import:
+
+- [Splitting data](#split-your-data-into-multiple-files)
+- [Import format](#choose-a-performant-import-format)
+
+{{site.data.alerts.callout_info}}
+If the import size is small (around 64-129 MiB), then you do not need to do anything to optimize performance. In this case, the import should run quickly, regardless of the settings.
+{{site.data.alerts.end}}
+
+## Split your data into multiple files
+
+Splitting the import data into multiple files can have a large impact on the import performance. We recommend splitting your data into as many files as there are nodes.
+
+For example, if you have a 3-node cluster, split your data into 3 files and import:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> IMPORT TABLE customers (
+        id UUID PRIMARY KEY,
+        name TEXT,
+        INDEX name_idx (name)
+)
+CSV DATA (
+    'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/customers.csv',
+    'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/customers_2.csv',
+    'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/customers_3.csv',
+);
+~~~
+
+CockroachDB imports the files that you give it, and does not further split them. For example, if you import one large file for all of your data, CockroachDB will process that file on one node– even if you have more nodes available. However, if you import two files (and your cluster has at least two nodes), each node will process a file in parallel. This is why splitting your data into as many files as you have nodes will dramatically decrease the time it takes to import data.
+
+{{site.data.alerts.callout_info}}
+If you split the data into _more_ files than you have nodes, it will not have a large impact on performance.
+{{site.data.alerts.end}}
+
+## Choose a performant import format
+
+Import formats do not have the same performance because of the way they are processed. Below, import formats are listed from fastest to slowest:
+
+1. [`CSV`](migrate-from-csv.html) or [`DELIMITED DATA`](import.html#delimited-data-files) (both have about the same import performance)
+1. [`AVRO`](migrate-from-avro.html)
+1. [`PGCOPY`](import.html)
```

Comment:
We don't mention PGCopy anywhere else in IMPORT -- should we mention it here? Or at least point out it's effectively a data only pgdump and pgdump is the format to use here?